WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -130,6 +158,25 @@ final public class InteractiveNotificationsManager : NSObject
     }
 
 
+    /// Replies to a comment
+    ///
+    /// - Parameters:
+    ///     - commentID: The comment identifier
+    ///     - siteID: The site identifier
+    ///     - content: The text for the comment reply
+    ///
+    private func replyToCommentWithCommentID(commentID: NSNumber, siteID: NSNumber, content: String) {
+        let context = ContextManager.sharedInstance().newDerivedContext()
+        let service = CommentService(managedObjectContext: context)
+
+        service.replyToCommentWithID(commentID, siteID: siteID, content: content, success: {
```

Comment:
Doh! I copied from the other actions and didn't notice the derived context. I think I'll switch them all to main context to avoid future problems even if they don't actually use core data yet.